code/modules/mob/emote.dm

```diff
@@ -31,8 +31,13 @@
 			to_chat(usr, "You are unable to emote.")
 		return
 
-	var/muzzled = istype(src.wear_mask, /obj/item/clothing/mask/muzzle)
-	if(m_type == 2 && muzzled) return
+	var/muzzled = is_muzzled()
+	if(muzzled)
+		var/obj/item/clothing/mask/muzzle/M = wear_mask
+		if(M.mute)
```

Comment:
This doesn't check if `m_type == 2`

Reply:
It does not, as in the comments, not all muzzles block sound.  This check assumed they did.

Now only muzzles that prevent you from making sounds block sounds.

If a muzzle has mute to 0, you can make a full range of noises.

This catches not emoting if they cannot speak
`if(m_type == 2 && !can_speak())`

Reply:
However, a muzzle with `mute` set, but against an emote with `m_type != 2`, would still block the emote - even though `m_type != 2` would have allowed the emote to function in the way this was implemented before